app/views/quotes/show.html.erb

```diff
@@ -0,0 +1 @@
+<%= @quote.author %> said, "<%= @quote.body %>", in the <%= @quote.location %>
```

Comment:
This is the same as the line in `app/views/quotes/index.html.erb` above. We can share the formatting here by using a partial http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials. This could allow us in the future to make changes in one place and have them apply to both pages. We'll likely want to do this when we're adding styles.

Having a partial like `app/views/quotes/_quote.html.erb` and then rendering that from both `show.html.erb` and `index.html.erb`. This pattern and naming is so common that rails supports a shortcut for it.

``` ruby
<%= render @quote %>
```

which does the same thing as

``` ruby
<%= render 'quote', quote: @quote %>
```